app/concepts/reservations/representer.rb

```diff
@@ -19,7 +19,16 @@ def basic
     end
 
     def extended
-      reservations
+      reservations.map do |reservation|
+        {
+            id: reservation.id,
+            status: reservation.status,
+            movie_id: reservation.screening_id,
+            ticket_desk_id: reservation.ticket_desk_id,
+            client_id: reservation.client_id,
+            tickets: reservation.tickets,
```

Comment:
I think you may want to use `Tickets::Representer` for this list of tickets as well, to have consistent way they are returned through whole application.